pkg/operator/ceph/client/controller_test.go

```diff
@@ -84,7 +84,7 @@ func TestGenerateClient(t *testing.T) {
 
 	client, caps := genClientEntity(p)
 	equal := bytes.Compare([]byte(client), []byte("client.client1"))
-	var res bool = equal == 0
+	res := equal == 0
 	assert.True(t, res)
```

Comment:
```suggestion
	assert.Equal(t, []byte(client), []byte("client.client1"))
```
I think this can be simplified and the intermediate var removed.

Reply:
thanks, that's a great suggestion to dry this up.